docassemble/AssemblyLine/data/questions/al_code.yml

```diff
@@ -74,20 +74,20 @@ objects:
 reconsider: True
 code: |
   al_court_bundle.insert(0, al_cover_page)
-  final_form_to_be_filed = al_court_bundle.as_pdf('final') 
+  al_final_form_to_file = al_court_bundle.as_pdf('final') 
```

Comment:
In most cases the court will want separate documents, not one big PDF. At least this is true for appeals court.

Reply:
Like all of the documents separately attached to the email? That's doable I guess, I'd just been avoiding changing as much outward-facing features as possible.